ios/RNCardVerify.m

```diff
@@ -10,8 +10,13 @@ - (void)setCallback:(RCTPromiseResolveBlock)resolve {
     }
 
     - (void)dismissView {
-        UIViewController *rootViewController = UIApplication.sharedApplication.delegate.window.rootViewController;
-        [rootViewController dismissViewControllerAnimated:YES completion:nil];
+        UIViewController *topViewController = [UIApplication sharedApplication].keyWindow.rootViewController;
```

Comment:
is there a reason to use this instead of the original `UIApplication.sharedApplication.delegate.window.rootViewController;`?

Reply:
no reason what so ever. i just copied pasted the code from `scan()`.

Reply:
I wrote it as topViewController because eventually it gets overwritten to actually be the topViewController